src/components/CompareResults/CompareResultsTableRow.tsx

```diff
@@ -25,7 +25,7 @@ function CompareResultsTableRow(props: ResultsTableRowProps) {
           className={`background-icon ${mode}-mode graph-icon-color`}
           aria-label="Graph link"
         >
-          <ShowChartIcon />
+          <TimelineIcon />
```

Comment:
added the change the graph icon 
<img width="932" alt="Screen Shot 2022-08-11 at 8 34 36 PM" src="https://user-images.githubusercontent.com/98465950/184264333-924734de-6c7f-46f3-98dc-bd4774fae5fc.png">

Reply:
<img width="882" alt="Screen Shot 2022-08-11 at 8 35 56 PM" src="https://user-images.githubusercontent.com/98465950/184264431-2cd5a0f8-8d6d-43ec-8431-916fa3305c2f.png">